### ISSUE TITLE
[Short.io] ...

### ISSUE THREAD

**Issue by iamdannywyatt** [raycast/extensions#2752]
# Extension – [Short.io](https://github.com/raycast/extensions/tree/72a9f2b7868c18a0b1693eeb372ac629422602dc/extensions/short-io/)
Author: @koinzhang

AxiosError: Request failed with status code 401

### Raycast version
Version: 1.39.1

I added the secret key and everything seems to be working, because when I perform the Shorten Link with Domain or the Shorten Link, it works. 

I had to create a test link first in order to being able to use the Search Links action.
So basically, when you first install the extension and try the Search Links, it will show you an error. 
After creating a new link using the extension, when you then go to Search Links, then you are able to see all your links.
I think this should be fixed.

Another bug:
I can't seem to change the default domain (the one with the yellow star). 
I press CMD+ Enter, but it doesn't change.

**Comment by koinzhang** [comment]
Hi @iamdannywyatt 
> After creating a new link using the extension, when you then go to Search Links, then you are able to see all your links.
I think this should be fixed.

I can't reproduce this problem, when there is no link, the search command will be displayed as follows.
<img width="400" alt="short-io 2022-09-03 at 00 27 05" src="https://user-images.githubusercontent.com/36128970/188198062-130d7d49-4dc9-4fff-85e0-8439d348d408.png">
> I can't seem to change the default domain (the one with the yellow star).
I press CMD+ Enter, but it doesn't change.

I'm sorry, I only have one domain so I can't test it, but I reviewed the code and the code logic is fine (there is a small bug with the display but it doesn't affect setting the default domain and I will submit a PR to fix it).
